toolkit/Makefile

```diff
@@ -97,7 +97,12 @@ TLS_KEY     ?=
 
 # Build defines
 DIST_TAG           ?= .cm1
-BUILD_NUMBER       ?= $(shell git rev-parse --short HEAD)
+GIT_REV            ?= $(shell git rev-parse --short HEAD)
```

Comment:
Is it possible use `BUILD_NUMBER` only and update it, in case it's empty after assigning it `$(shell git rev-parse --short HEAD)`?

Reply:
Yup, that works as well. I updated the PR.